Cargo.toml

```diff
@@ -660,15 +656,11 @@ bp-asset-hub-rococo = { path = "bridges/chains/chain-asset-hub-rococo", default-
 bp-asset-hub-westend = { path = "bridges/chains/chain-asset-hub-westend", default-features = false }
 bp-beefy = { path = "bridges/primitives/beefy", default-features = false }
 bp-bridge-hub-cumulus = { path = "bridges/chains/chain-bridge-hub-cumulus", default-features = false }
-bp-bridge-hub-kusama = { default-features = false, path = "bridges/chains/chain-bridge-hub-kusama" }
-bp-bridge-hub-polkadot = { path = "bridges/chains/chain-bridge-hub-polkadot", default-features = false }
 bp-bridge-hub-rococo = { path = "bridges/chains/chain-bridge-hub-rococo", default-features = false }
 bp-bridge-hub-westend = { path = "bridges/chains/chain-bridge-hub-westend", default-features = false }
 bp-header-chain = { path = "bridges/primitives/header-chain", default-features = false }
-bp-kusama = { default-features = false, path = "bridges/chains/chain-kusama" }
 bp-messages = { path = "bridges/primitives/messages", default-features = false }
 bp-parachains = { path = "bridges/primitives/parachains", default-features = false }
-bp-polkadot = { default-features = false, path = "bridges/chains/chain-polkadot" }
 bp-polkadot-bulletin = { path = "bridges/chains/chain-polkadot-bulletin", default-features = false }
```

Comment:
@antkve I think we will need to do the same for `bp-polkadot-bulletin`, but let's wait until the time comes (related to PoP bridge).

Reply:
Yes, probably. But I think we can do it now.

Reply:
> Yes, probably. But I think we can do it now.

Ok, ok, then let's do it :)